x-pack/common/constants/license_types.ts

```diff
@@ -4,15 +4,25 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+export const LICENSE_TYPE_OSS = 'oss';
```

Comment:
Just out of curiosity, how is it possible for there to be an OSS license type if the OSS distribution ships without X-Pack?

Reply:
I don't think it is possible 🙂 I added this for the sake of completeness, and because I saw `oss` included in a [similar export from beats management](https://github.com/elastic/kibana/blob/master/x-pack/plugins/beats_management/common/constants/security.ts#L10)

Reply:
👍 I think it might be worth checking in with them and making sure that wasn't a mistake, so we can avoid propagating it if it was.

Reply:
I just checked with @mattapperson and `oss` was included intentionally at the time to support a "read only" mode if the user downgrades. however, plans for that support haven't really moved, so it's not necessarily to declare `oss` here for the time being. we can always add it when it's actually needed. thanks for calling this out, @cjcenizal, I've pushed an update that removes the `oss` declaration